agent/config/parse_linux.go

```diff
@@ -69,8 +69,8 @@ func parseGMSACapability() BooleanDefaultFalse {
 	return BooleanDefaultFalse{Value: ExplicitlyDisabled}
 }
 
-func parseFSxWindowsFileServerCapability() BooleanDefaultFalse {
-	return BooleanDefaultFalse{Value: ExplicitlyDisabled}
+func parseFSxWindowsFileServerCapability() BooleanDefaultTrue {
+	return BooleanDefaultTrue{Value: ExplicitlyDisabled}
```

Comment:
same comment as for config_unix, I would just leave these unchanged since we want the default to be false for Linux.

Reply:
synced offline, this is because `FSxWindowsFileServerCapable` is a shared config type between both Windows, Linux platforms.

We've added tests that validate that this config type is true for Windows and false for everything else. Resolving related threads below too.

Reply:
Same as above.